GRPCServiceA/pkg/user/transport.go

```diff
@@ -0,0 +1,104 @@
+package user
+
+import (
+	"context"
+
+	ent "github.com/coding-kiko/GoKit-Project-Bootcamp/GRPCServiceA/pkg/entities"
+	erro "github.com/coding-kiko/GoKit-Project-Bootcamp/GRPCServiceA/pkg/errors"
+	"github.com/coding-kiko/GoKit-Project-Bootcamp/GRPCServiceA/pkg/user/proto"
+	"github.com/go-kit/kit/log"
+	gt "github.com/go-kit/kit/transport/grpc"
+)
+
+type gRPCServer struct {
+	getUser    gt.Handler
+	createUser gt.Handler
+	proto.UnimplementedUserServicesServer
+}
+
+func NewGRPCServer(endpoints Endpoints, logger log.Logger) proto.UserServicesServer {
```

Comment:
you're receiving a logger but you don't use it, delete it, or use clusters to log from endpoints and coders.